### PR TITLE
Make mod_version multitenancy ready

### PR DESCRIPTION
**Pull request by NelsonVides**
As in the title

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/71374) / small_tests / b63ea3c918a8d258dc77ea6f4a9b1d0f161d208e
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3256/71374/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3256/71374/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-09_16.29.34/lib.mongooseim.logs/run.2021-09-09_16.29.34/suite.log.html)

File: src/mod_version.erl [hunk 1]
```diff
@@ -31,13 +38,13 @@ config_spec() ->
                 }
       }.
 
--spec process_iq(jid:jid(), jid:jid(), mongoose_acc:t(), jlib:iq()) ->
+-spec process_iq(jid:jid(), jid:jid(), mongoose_acc:t(), jlib:iq(), any()) ->
```
Comment (chrzaszcz) [inline src/mod_version.erl:41]:
```suggestion
-spec process_iq(mongoose_acc:t(), jid:jid(), jid:jid(), jlib:iq(), any()) ->
```

**Review by chrzaszcz — COMMENTED** [review]
Code looks good in general, I have one comment.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/71618) / small_tests / d5a2bcc732fe8c491b1e14ea7bec0ff42e03c078
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3256/71618/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3256/71618/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-10_07.57.07/lib.mongooseim.logs/run.2021-09-10_07.57.07/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Good!

**Pull request merged**